packages/hmr-plugin/src/internal.ts

```diff
@@ -0,0 +1,51 @@
+import { NgModuleRef } from '@angular/core';
+import { InternalStateOperations } from '@ngxs/store';
+
+import { HmrNgxsStoreOnDestroyFn, HmrNgxsStoreOnInitFn, NGXS_HMR, NgxsStoreSnapshot } from './symbols';
+import { StateOperations } from '@ngxs/store/src/internal/internals';
+
+export function hmrDoBootstrap<T = NgxsStoreSnapshot>(ref: NgModuleRef<any>): NgModuleRef<any> {
+  const hmrNgxsStoreOnInitFn: HmrNgxsStoreOnInitFn<T> = ref.instance[NGXS_HMR.hmrNgxsStoreOnInit];
+
+  if (typeof hmrNgxsStoreOnInitFn === 'function') {
+    const stateOperation: StateOperations<NgxsStoreSnapshot> = getStateOperations(ref);
+    stateOperation && hmrNgxsStoreOnInitFn(getStateOperations(ref), getStateFromHmrStorage());
```

Comment:
```suggestion
    stateOperation && hmrNgxsStoreOnInitFn(stateOperation, getStateFromHmrStorage());
```
use the variable from above... as mentioned in another comment this should rather be a `StateContext` reference too.

Reply:
Fixed in my recent commit to this branch.